omnigibson/robots/robot_base.py

```diff
@@ -232,14 +241,18 @@ def _load_sensors(self):
                             if self._obs_modalities == "all"
                             else sensor_cls.all_modalities.intersection(self._obs_modalities)
                         )
+                    # If the modalities list is empty, don't import the sensor.
+                    if not sensor_kwargs["modalities"]:
```

Comment:
is this for perf reasons?

Reply:
I believe so, do you think this would cause any problems?

Reply:
It is, yes. Avoids a bunch of replicator stuff running for no reason